Userland/Libraries/LibXML/Parser/Parser.cpp

```diff
@@ -409,6 +409,7 @@ ErrorOr<void, ParseError> Parser::parse_standalone_document_decl()
     TRY(expect("standalone"sv));
     auto accept = accept_rule();
 
+    TRY(parse_eq());
```

Comment:
Please move this to a separate commit.

Reply:
Ok, I will close this PR and start a new one with this change.